### ISSUE TITLE
Get-Help Invoke-Build -Full doesn't work

### ISSUE THREAD

**Issue by NReilingh** [nightroman/Invoke-Build#41]
I've installed IB from PSGallery and am trying to view the xml-based help with `Get-Help Invoke-Build -Full`. I am on PS 5.1.

I installed IB with powershellget, so the module is in \Program Files\WindowsPowerShell\Modules\. In a fresh console, `Get-Help *build*` returns about_InvokeBuild, and doing `Import-Module InvokeBuild` first returns two additional rows for the Invoke-Build[s] aliases.

![screen shot 2016-12-17 at 6 27 51 pm](https://cloud.githubusercontent.com/assets/2458645/21290387/94858bae-c486-11e6-8b05-ed0f63d90808.png)

`Get-Help Invoke-Build -Full` just returns those two alias rows, instead of what's described in README.md. Dot-sourcing Invoke-Build does then add help for the helper functions, but doesn't change the behavior for Invoke-Build.

Based on [Naming Help Files](https://msdn.microsoft.com/en-us/library/dn423135(v=vs.85).aspx), it seems that XML-based help may only be possible for actual functions.

**Comment by nightroman** [comment]
It looks like `Get-Help` does not resolve aliases in this case, unfortunately. Currently, I have no idea how to solve this. All I can think of right now is to use `Get-Help <path>\Invoke-Build.ps1` and document this in README. Any better ideas?

**Comment by nightroman** [comment]
I played with aliases to scripts a bit more and found that they are resolved by `Get-Help` if scripts are in the path. I think it is a PS defect/bug.

Can you try to add the module directory to the path and invoke `Get-Help invoke-build`?

**Comment by nightroman** [comment]
I opened the PowerShell bug. I propose to document this in README, see below, and close this issue.

----

## Getting help

If you are using the module (see [#2899]) or the script is not in the path
then use the full path to *Invoke-Build.ps1* instead of *Invoke-Build* in
the below commands.

[#2899]: https://github.com/PowerShell/PowerShell/issues/2899

In order to get help for the engine, invoke:

    help Invoke-Build -full

In order to get help for internal commands:

    . Invoke-Build
    help task -full
    help exec -full
    ...

**Comment by NReilingh** [comment]
Yup, adding the module directory to path does it. Thanks for the hint about specifying the direct path to the script for Get-Help as well. Proposed readme update looks good.

**Comment by nightroman** [comment]
Updated README in by https://github.com/nightroman/Invoke-Build/commit/0967cd7f13d2c34b2838f37edde84976dceb1296.

**Comment by NReilingh** [comment]
I kept running into this periodically since I don't use InvokeBuild (or even PowerShell) every day, so what I finally did was programmatically add the module to my path variable in my CurrentUserAllHosts profile:

```powershell
Import-Module InvokeBuild
$env:Path = "$env:Path;" + (Split-Path (Get-Alias Invoke-Build).Definition)
```

I believe this will stay correct even as the version number for the InvokeBuild module increments (which would change the path). Doing an Import-Module in my profile isn't ideal, but InvokeBuild's module file is so lightweight that I don't expect it to be a performance annoyance (certainly not to the degree of annoyance that typing `man (gal (gal ib).def<tab>).def<tab>` is).

It occurs to me that to avoid importing the module, I could also do:

```powershell
$env:Path = "$env:Path;" + (Get-Module InvokeBuild -ListAvailable | Select-Object -First 1 -Property Path | Split-Path)
```

But I have no idea if this is meaningfully faster.
